examples/gallery/README.md

```diff
@@ -34,7 +34,7 @@ To test all examples with the local version, run:
 
 ```bash
 # rebuild bundle
-cd ../../modules/lite && npm run build && cd -
+cd ../../modules/main && npm run build-bundle && cd -
```

Comment:
Run prepublish which will give you both dust and dist.min. dist is easier to use when debugging.